composition/fetch_definition.go

```diff
@@ -69,14 +71,23 @@ type FetchDefinition struct {
 	CacheStrategy          CacheStrategy
 	ServiceDiscoveryActive bool
 	ServiceDiscovery       servicediscovery.ServiceDiscovery
+	Priority               int
 }
 
 // Creates a fetch definition
 func NewFetchDefinition(url string) *FetchDefinition {
-	return NewFetchDefinitionWithResponseProcessor(url, nil)
+	return NewFetchDefinitionWithResponseProcessorAndPriority(url, nil, DefaultPriority)
+}
+
+func NewFetchDefinitionWithPriority(url string, priority int) *FetchDefinition {
+	return NewFetchDefinitionWithResponseProcessorAndPriority(url, nil, priority)
 }
 
 func NewFetchDefinitionWithErrorHandler(url string, errHandler ErrorHandler) *FetchDefinition {
+	return NewFetchDefinitionWithErrorHandlerAndPriority(url, errHandler, DefaultPriority)
+}
+
+func NewFetchDefinitionWithErrorHandlerAndPriority(url string, errHandler ErrorHandler, priority int) *FetchDefinition {
```

Comment:
Maybe we should use a [fluent interface](https://de.wikipedia.org/wiki/Fluent_Interface) style api, because the method names are becoming quite long. But this is no blocker for this PR for me